lib/tasks/argo.rake

```diff
@@ -18,6 +18,7 @@ end
 task :default => [:ci]
 
 task :ci do
+  raise 'NEVER RUN CI ON PROD' if ENV['RAILS_ENV'] =~ /production/i
   ENV['RAILS_ENV'] = 'test'
```

Comment:
AFAIK, the RAILS_ENV doesn't necessarily imply that production configs will use non-prod systems and services.  If they can be decoupled, as we have discussed for staging, then this safeguard helps.